### PR TITLE
Remove visual style for title and message label font

### PR DESCRIPTION
**Pull request by twiedow**
Use NSAttributedString instead.

### CODE REVIEW

**Comment by twiedow** [comment]
I rebased my branch to your latest master. Please review the changes.

File: Source/AlertControllerView.swift [hunk 1]
```diff
@@ -45,6 +45,12 @@ class AlertControllerView: UIView {
         return lastElement.frame.maxY + self.visualStyle.contentPadding.bottom
     }
 
+    convenience init() {
+        self.init(frame: CGRectZero)
```
Comment (sberrevoets) [inline Source/AlertControllerView.swift]: Could you change this to `.zero`?

**Comment by sberrevoets** [comment]
Left 1 comment, other than that it looks good. Thanks for keeping it updated :100:

**Comment by twiedow** [comment]
Thanks for reviewing, done :-)

**Pull request merged**